tgui/packages/tgui/interfaces/AresInterface.jsx

```diff
@@ -279,6 +280,19 @@ const MainMenu = (props) => {
                 onClick={() => act('page_access')}
               />
             </Stack.Item>
+            <Stack.Item>
```

Comment:
It is a bit weird that working joes cannot access core security protocols in this interface, but they can in their maintenance interface.